x-pack/plugins/security_solution/public/detections/containers/detection_engine/alerts/use_cases_from_alerts.tsx

```diff
@@ -29,7 +29,7 @@ export const useCasesFromAlerts = ({ alertId }: { alertId: string }): CasesFromA
     setLoading(true);
     const fetchData = async () => {
       try {
-        const casesResponse = await getCaseIdsFromAlertId({ alertId, owner: [APP_ID] });
+        const casesResponse = await getCaseIdsFromAlertId({ alertId, owner: [APP_UI_ID] });
```

Comment:
@XavierM changing the _owner_ id everywhere will not cause problems? Will it get the same cases as before the change? Maybe we can keep using `APP_ID` for the _owner_.

Reply:
that's my mistake good catch